src/components/Title/index.js

```diff
@@ -18,8 +18,12 @@ Title.defaultProps = {
 
 Title.propTypes = {
   children: PropTypes.node.isRequired,
-  theme: PropTypes.oneOf([Themes.red, Themes.neonCarrot, Themes.greyishBlue])
-    .isRequired,
+  theme: PropTypes.oneOf([
+    Themes.red,
+    Themes.blue,
+    Themes.neonCarrot,
+    Themes.greyishBlue,
+  ]).isRequired,
```

Comment:
again, just my opinion, but we could try to keep this sorted

Reply:
Yes